app/views/liquids/_medium.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class='ltag__link'>
   <a href='<%= response[:url] %>' class='ltag__link__link'>
     <div class='ltag__link__pic'>
-      <img src='<%= response[:author_image] %>' alt='<%= response[:author] %>'/>
+      <img src='<%= response[:author_image] %>' alt='<%= response[:author] %>' />
     </div>
   </a>
   <a href='<%= response[:url] %>' class='ltag__link__link'>
     <div class='ltag__link__content'>
       <h2><%= response[:title] %></h2>
-      <h3><%= response[:author] %> ・ <%= response[:reading_time] %></h3>
+      <h3><%= response[:author] %> ・ <%= response[:publication_date] %> ・ <%= response[:reading_time] %></h4>
```

Comment:
it would be awesome if you could use the `<time>` element here. 

Basically:

```html
<time datetime="YYYY-MM-DD"><%= response[:publication_date] %></time>
```

you can see more details here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

I suggest, instead of using `YYYY-MM-DD` in the `datetime` attribute, to use the full timestamp string, eg. `2019-12-03T15:43:04.151Z`, browsers add helpful tooltips:

![Screenshot 2019-12-04 at 1 14 57 PM](https://user-images.githubusercontent.com/146201/70141854-34820e00-1698-11ea-8c36-cfd1f4a623c3.png)